docs/hub/datasets-adding.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Adding New Datasets
+---
+
+<h1>Adding New Datasets</h1>
+
+There are two different kinds of datasets on the Hub: **community datasets** and **canonical datasets**.
```

Comment:
i would de-emphasize canonical datasets to the maximum as we expect most of the new datasets to be "community"